src/main/java/hng_java_boilerplate/product/service/ProductServiceImpl.java

```diff
@@ -29,4 +31,25 @@ public Page<Product> productsSearch(String name, String category, Double minPric
         }
         return productRepository.searchProducts(name, category, minPrice, maxPrice, pageable);
     }
+
+
+    @Override
+    public Product addProduct(ProductDTO productDTO, String userId) {
```

Comment:
PR is "update product", issue and endpoint is "add product". Not the same thing

Reply:
I am trying to change the branch, But I can't